### ISSUE TITLE
Use <strong> tag inside standfirst

### ISSUE THREAD

**Issue by SiAdcock** [guardian/dotcom-rendering#14179]
From the [SEO audit](https://drive.google.com/file/d/1XS9QP1HkjGkIJ4ehcoSVatVqWJQ8NhNX/view?usp=sharing)

> Change the HTML markup for your articles’ lead-in content. Purely for SEO an `<h2>` heading tag should be implemented to ensure Google pays the most attention to the text, but this poses accessibility challenges. Therefore I recommend using a `<strong>` HTML tag in addition to the `<p>` tag.

> This also applies when the standfirst content is composed of bullet points – additional emphasis can be added there too with the `<strong>` HTML tag.

> The visual appearance of the standfirst text doesn’t need to change. The addition of semantic HTML for emphasis is purely to indicate to Google that the text carries more weight than the rest of the article’s content. The standfirst text is already visually emphasised, so also emphasising it with the `<strong>` HTML tag would align with the text’s visual appearance.
